src/Test.UnitTests.Sarif/Visitors/InsertOptionalDataVisitorTests.cs

```diff
@@ -48,7 +48,7 @@ protected override string ConstructTestOutputFromInputResource(string inputResou
             // to the expected output.
             string enlistmentRoot = GitHelper.Default.GetRepositoryRoot(Environment.CurrentDirectory, useCache: false);
 
-            if (inputResourceName == "CoreTests-Relative.sarif")
+            if (inputResourceName == "Rel.sarif")
```

Comment:
>Rel [](http://example.com/codeflow?start=38&length=3)

verbose "Relative" here is easier to understand imo. Why was it shortened?

Reply:
I mentioned this elsewhere will add the note here too. thanks!

Reply:
for windows, default is something like
`C:\Users\%USERNAME%\AppData\Roaming`
if it related to the length of username as well, then a longer alias person will be easier to have problem.